src/Service.Host/package.json

```diff
@@ -84,6 +84,7 @@
         "es6-promise": "^4.1.1",
         "eslint": "^5.15.1",
         "history": "^4.7.2",
+        "lodash": "^4.17.11",
```

Comment:
is lodash actually being used?

Reply:
ah dear, it was now it's not - will remove